package.json

```diff
@@ -22,7 +22,7 @@
     "npm-watch": "0.7.0",
     "postcss": "8.4.31",
     "postcss-cli": "8.3.1",
-    "prettier": "2.1.1",
+    "prettier": "^2.1.1",
```

Comment:
this is the reason linting is breaking - you'll need to remove the ^ to pin the version!

Reply:
Ahhhh, thank you! Fixed :+1: